app.js

```diff
@@ -1,4 +1,48 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+
+var express = require('express');
+var trucks = require('./trucks')
+
+var app = express();
+
+app.use(express.static('public'));
+
+app.get('/', function(request, response){
```

Comment:
This is unnecessary. By using the express.static middle-ware and having an index.html file, the route will be handled before it hits this code.